packages/charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -1,90 +1,248 @@
+            <div>
+              <strong>{data!.chartTitle}</strong>
+            </div>
+          )}
+          {showRatio && (
+            <div>
+              <strong>{data!.chartData![0].data}</strong>/{total}
+            </div>
+          )}
+          {showNumber && (
+            <div>
+              <strong>{data!.chartData![0].data}</strong>
             </div>
-          );
-        })}
-        {this._renderLegend && <Legends legends={legends} />}
+          )}
+        </div>
+        <svg className={this._classNames.chart}>{bars}</svg>
       </div>
     );
   }
 
-  private _getLegendData = (data: IChartProps[], hideRatio: boolean[]): ILegend[] => {
+  private _refCallback(element: SVGGElement, legendTitle: string): void {
+    this.state.refArray.push({ legendText: legendTitle, refElement: element });
+  }
+
+  private _adjustProps = (): void => {
+    const { theme, className, styles, width, barHeight } = this.props;
+    this._classNames = getClassNames(styles!, {
+      legendColor: this.state.color,
+      theme: theme!,
+      width: width,
+      className,
+      barHeight: barHeight
+    });
+  };
+
+  private _onHover(customMessage: string, pointData: number, color: string): void {
+    if (!this.state.isCalloutVisible || this.state.legendSelected !== customMessage) {
+      const refArray = this.state.refArray;
+      const currentHoveredElement = refArray.find(
+        (eachElement: IRefArrayData) => eachElement.legendText === customMessage
+      );
+      this.setState({
+        isCalloutVisible: true,
+        refSelected: currentHoveredElement!.refElement,
+        legendSelected: customMessage,
+        dataForHoverCard: pointData,
+        color: color
+      });
+    }
+  }
+
+  private _getLegendData = (data: IChartProps[], hideRatio: boolean[], palette: IPalette): JSX.Element => {
+    const defaultPalette: string[] = [palette.blueLight, palette.blue, palette.blueMid, palette.red, palette.black];
     const actions: ILegend[] = [];
     data.map((singleChartData: IChartProps, index: number) => {
       if (singleChartData.chartData!.length < 3) {
-        if (hideRatio && hideRatio[index]) {
-          singleChartData.chartData!.map((dataPoint: IChartDataPoint) => {
-            const action: ILegend = {
-              title: dataPoint.legend!,
-              color: dataPoint.color!
+        const hideNumber = hideRatio[index] === undefined ? false : hideRatio[index];
+        if (hideNumber) {
+          singleChartData.chartData!.map((point: IChartDataPoint) => {
+            const color: string = point.color ? point.color : defaultPalette[Math.floor(Math.random() * 4 + 1)];
+            const pointData = point.data ? point.data : 0;
+            const legend: ILegend = {
+              title: point.legend!,
+              color: color,
+              action: () => {
+                this._onHover(point.legend!, pointData, color);
+              },
+              hoverAction: () => {
```

Comment:
There are some type errors here that are causing the build to fail:

  Object literal may only specify known properties, and 'hoverAction' does not exist in type 'ILegend'.
203               hoverAction: () => {
                  ~~~~~~~~~~~~~~~~~~~~
204                 this._onHover(point.legend!, pointData, color);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
205               },
    ~~~~~~~~~~~~~~~
src/components/StackedBarChart/MultiStackedBarChart.base.tsx:223:13 - error TS2322: Type '{ title: string; color: string; action: () => void; hoverAction: () => void; onMouseOutAction: ()...' is not assignable to type 'ILegend'.
  Object literal may only specify known properties, and 'hoverAction' does not exist in type 'ILegend'.
223             hoverAction: () => {
                ~~~~~~~~~~~~~~~~~~~~
224               this._onHover(point.legend!, pointData, color);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
225             },
    ~~~~~~~~~~~~~
src/components/StackedBarChart/StackedBarChart.base.tsx:105:57 - error TS6133: 'isMultiStackedBarChart' is declared but its value is never read.
105     const { theme, className, styles, width, barHeight, isMultiStackedBarChart } = this.props;
                                                            ~~~~~~~~~~~~~~~~~~~~~~
src/components/StackedBarChart/StackedBarChart.base.tsx:136:9 - error TS2322: Type '{ title: string; color: string; action: () => void; hoverAction: () => void; onMouseOutAction: ()...' is not assignable to type 'ILegend'.
  Object literal may only specify known properties, and 'hoverAction' does not exist in type 'ILegend'.
136         hoverAction: () => {
            ~~~~~~~~~~~~~~~~~~~~
137           this._onHover(point.legend!, pointData, color);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
138         },

Reply:
This is because I haven't updated my branch after you merged the PR this morning. Will take care of this.